internal/types.go

```diff
@@ -54,6 +54,9 @@ const (
 
 	// View reltype
 	View
+
+	//Partition reltype
+	Partition
```

Comment:
That's misleading. It's used for partitioned tables, not partitions, so rename it, please.

Reply:
Done